packages/base/express/graphql.js

```diff
@@ -52,8 +52,9 @@ module.exports = (
             return createContext()
           }
           const cookies = cookie.parse(cookiesRaw)
-          const sid = cookieParser.signedCookie(
-            cookies['connect.sid'],
+          const authCookie = cookies['connect.sid']
+          const sid = authCookie && cookieParser.signedCookie(
+            authCookie,
             process.env.SESSION_SECRET
```

Comment:
This has nothing to do with notifications or subscriptions, but a auth problem should be fixed as soon as spotted.